pkg/controller/statefulset/stateful_set_utils.go

```diff
@@ -180,6 +182,30 @@ func updateIdentity(set *appsv1alpha1.StatefulSet, pod *v1.Pod) {
 	pod.Labels[apps.StatefulSetPodNameLabel] = pod.Name
 }
 
+// isRunningAndAvailable returns true if pod is in the PodRunning Phase,
+// and it has a condition of PodReady for a minimum of minReadySeconds.
+// return true if it's available
+// return false with zero means it's not ready
+// return false with a positive value means it's not available and should recheck with that time
+func isRunningAndAvailable(pod *v1.Pod, minReadySeconds int32) (bool, time.Duration) {
```

Comment:
I would suggest the first returned boolean indicates "RunningAndRunning" and the 2nd returned value indicates the wait time. I think this makes the code easier to understand.

Reply:
> RunningAndRunning

what is "RunningAndRunning"? Currently, it means if it's "RunningAndAvailable"

Reply:
Sorry, typo, I mean RunningAndReady.